public/css/style.css

```diff
@@ -0,0 +1,177 @@
+@font-face {
+  font-family: "Gotham";
+  src: url("https://storage.googleapis.com/hackgt/fonts/Gotham-Medium.otf") format('opentype');
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Gotham Book";
+  src: url("https://storage.googleapis.com/hackgt/fonts/Gotham-Book.otf") format('opentype');
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Space Mono";
+  src: url("https://storage.googleapis.com/hackgt/fonts/SpaceMono-Regular.ttf") format('opentype');
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Space Mono";
+  src: url("https://storage.googleapis.com/hackgt/fonts/SpaceMono-Bold.ttf") format('opentype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+/* CSS variables for colors */
+:root {
+  --bg-color: #FFFFFE;
+  --text-color: #4A4A4A;
+  --dark-blue: #036BFF;
+  --light-blue: #8FBEFF;
+  --blur: rgba(255, 255, 255, 0.6);
+}
+
+/* Set backgrounds */
+html::before {
+  background-image: url("https://horizons.hack.gt/assets/Blobs.svg");
+  background-size: 100% auto;
+  background-position: top center;
+  background-repeat: no-repeat;
+  opacity: 0.4;
+
+  content: "";
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+}
+
+#sidebar-nav {
+  background: none;
+}
+section.main {
+  background-color: var(--blur);
+}
+
+html, body, #sidebar-nav, section.main {
+  color: var(--text-color);
+  font-family: "Space Mono", sans-serif;
+}
+
+/* Hide text and use image instead in the header */
+.alignleft {
+  display: block;
+  background-image: url("https://horizons.hack.gt/assets/horizons-logo.png");
+  background-size: contain;
+  background-position: top center;
+  background-repeat: no-repeat;
+  height: 75px;
+
+  color: rgba(0, 0, 0, 0);
+  -ms-user-select: none;
+  -moz-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+}
+#sidebar > h1 {
+  margin-bottom: -10px;
+}
+
+#sidebar-nav > a, #sidebar, .editor-toolbar > a {
+  color: var(--text-color);
+  border: none;
+  text-decoration: initial;
+  font-family: "Space Mono";
+  font-weight: bold;
+}
+.editor-toolbar > a {
+  color: var(--text-color) !important;
+}
+
+.editor-toolbar a.active, .editor-toolbar a:hover {
+  border-color: #4C5D67;
+  background-color: rgba(196, 251, 245, 0.2);
+  outline: none;
+}
+#sidebar-nav > span.divider, section.main, table thead {
+  border-color: var(--light-blue);
+}
+a {
+  color: var(--text-color);
+  text-decoration: none;
+  border-bottom: 1px dotted;
+  cursor: pointer;
+}
+a:hover {
+  border-bottom-style: solid;
+}
+
+::-webkit-input-placeholder { /* WebKit, Blink, Edge */
+  color: var(--text-color);
+}
+
+#email-rendered {
```

Comment:
I don't know why we have these styles and the form styles – can we get rid of useless CSS styles? I'm sure there's an auditor somewhere that'll tell you which styles a page is using